include/sys_clock.h

```diff
@@ -53,6 +53,8 @@ extern void z_enable_sys_clock(void);
 /* number of nanoseconds per second */
 #define NSEC_PER_SEC ((NSEC_PER_USEC) * (USEC_PER_MSEC) * (MSEC_PER_SEC))
 
+#define k_msleep(ms) k_sleep(ms)
```

Comment:
I think it would be better to document this function and add it to our APIs alongside where k_sleep() is defined (which IIRC is in kernel.h). at minimum I think doxygen is needed.

Reply:
It gets that in an upcoming patch in the series, where it becomes a real function (because k_sleep() now takes a k_timeout_t).  I put the placeholders here just to grease the refactoring so everything bisects without weird splitting of features.